src/utils/Shell.ts

```diff
@@ -105,12 +105,39 @@ class ZSH extends Shell {
     }
 }
 
+class Cmd extends Shell {
+    get executableName() {
+        return "cmd.exe";
+    }
+
+    get configFiles() {
+        return [
+        ];
+    }
+
+    get noConfigSwitches() {
+        return [];
+    }
+
+    get preCommandModifiers(): string[] {
+        return [];
+    }
+
+    get historyFileName(): string {
+        return "";
+    }
+}
+
 const supportedShells: Dictionary<Shell> = {
     bash: new Bash(),
     zsh: new ZSH() ,
+    'cmd.exe': new Cmd()
 };
 
 const shell = () => {
+    if (!process.env.SHELL) {
+        process.env.SHELL = 'C:\\Windows\\System32\\cmd.exe';
+    }
```

Comment:
Debug leftover; just drop this.

Although we'll probably better add Windows detection and fallback to `cmd.exe` instead of `/bin/bash` on Windows below.